assert/assertions.go

```diff
@@ -845,19 +845,26 @@ func containsElement(list interface{}, element interface{}) (ok, found bool) {
 		}
 	}()
 
-	if listKind == reflect.String {
+	switch listKind {
+	case reflect.String:
 		elementValue := reflect.ValueOf(element)
 		return true, strings.Contains(listValue.String(), elementValue.String())
-	}
-
-	if listKind == reflect.Map {
+	case reflect.Map:
 		mapKeys := listValue.MapKeys()
 		for i := 0; i < len(mapKeys); i++ {
 			if ObjectsAreEqual(mapKeys[i].Interface(), element) {
 				return true, true
 			}
 		}
 		return true, false
+	case reflect.Slice:
+		elementValue := reflect.ValueOf(element)
+		if elementValue.Kind() != reflect.Slice ||
+			elementValue.Type().Elem().Kind() != reflect.Uint8 ||
+			listType.Elem().Kind() != reflect.Uint8 {
+			break
+		}
```

Comment:
This condition looks pretty complex. I think it would be super helpful if a comment explains the intent. Ex:   Why are we discarding `uint8`?

Reply:
@arjunmahishi Inverting the `if` made things a lot clearer. Also added a brief comment. Please let me know if it could still be clearer